src/amo/components/PermissionsCard/index.js

```diff
@@ -0,0 +1,280 @@
+    // which can be found in
+    // https://hg.mozilla.org/mozilla-central/raw-file/tip/browser/locales/en-US/chrome/browser/browser.properties
+    const permissionStrings = {
+      bookmarks: i18n.gettext('Read and modify bookmarks'),
+      browserSettings: i18n.gettext('Read and modify browser settings'),
+      browsingData: i18n.gettext('Clear recent browsing history, cookies, and related data'),
+      clipboardRead: i18n.gettext('Get data from the clipboard'),
+      clipboardWrite: i18n.gettext('Input data to the clipboard'),
+      devtools: i18n.gettext('Extend developer tools to access your data in open tabs'),
+      downloads: i18n.gettext('Download files and read and modify the browser’s download history'),
+      'downloads.open': i18n.gettext('Open files downloaded to your computer'),
+      find: i18n.gettext('Read the text of all open tabs'),
+      geolocation: i18n.gettext('Access your location'),
+      history: i18n.gettext('Access browsing history'),
+      management: i18n.gettext('Monitor extension usage and manage themes'),
+      // In Firefox the following message replaces the name "Firefox" with the
+      // current brand name, e.g., "Nightly", but we do not need to do that.
+      nativeMessaging: i18n.gettext('Exchange messages with programs other than Firefox'),
+      notifications: i18n.gettext('Display notifications to you'),
+      pkcs11: i18n.gettext('Provide cryptographic authentication services'),
+      proxy: i18n.gettext('Control browser proxy setting'),
+      privacy: i18n.gettext('Read and modify privacy settings'),
+      sessions: i18n.gettext('Access recently closed tabs'),
+      tabs: i18n.gettext('Access browser tabs'),
+      tabHide: i18n.gettext('Hide and show browser tabs'),
+      topSites: i18n.gettext('Access browsing history'),
+      unlimitedStorage: i18n.gettext('Store unlimited amount of client-side data'),
+      webNavigation: i18n.gettext('Access browser activity during navigation'),
+      allUrls: i18n.gettext('Access your data for all websites'),
+      wildcard: (domain) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data for sites in the %(domain)s domain'),
+          { domain }
+        );
+      },
+      tooManyWildcards: (count) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data in %(count)s other domains'),
+          { count: i18n.formatNumber(count) },
+        );
+      },
+      oneSite: (site) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data for %(site)s'),
+          { site }
+        );
+      },
+      tooManySites: (count) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data on %(count)s other sites'),
+          { count: i18n.formatNumber(count) },
+        );
+      },
+    };
+
+    // Classify a permission as a host permission or a regular permission.
+    function classifyPermission(permission) {
```

Comment:
I'd like to be able to write unit tests for this, but wasn't sure how to access it directly from a test as it's inside a component. I didn't want to export it as it's not used anywhere else. Perhaps it would be better to move this to a helper module where it could be more easily tested?